src/hooks/Auth/useCSSVariables.ts

```diff
@@ -0,0 +1,22 @@
+export default (): {
```

Comment:
This file should not be under the Auth folder.

Reply:
oh!